MountSionCBSSecondary/SchoolWebsite
# Fix logo typo

# Description

Changed logo to this
![school logo](https://user-images.githubusercontent.com/31955648/50292959-6b045000-046a-11e9-9b71-d0470a638b15.jpg)


## What

Logo is changed
## Why

There was a typo in logo
## How

Remade the logo in ps 
## Verification Steps

open the website and see if worked. I made a second pr because I used wrong image, that's why you check before you make a pr.

# Checklist:
